src/simsopt/mhd/vmec.py

```diff
@@ -752,7 +752,11 @@ def run(self):
 
             # Delete the previous output file, if desired:
             for filename in self.files_to_delete:
-                os.remove(filename)
+                try:
```

Comment:
you can also check if the file exists and if it is not the case you can thrown an error.

Reply:
The way I've done it is similar to how it's done elsewhere in the same script, and it seems more pythonic to ask for forgiveness rather than permission.
There may also be practical reasons for doing it this way: the file probably ceased to exist just before os.remove tried to remove it (if it didn't exist recently, it'd not be in the files_to_delete list), so checking if it exists beforehand may not be reliable, since it may get deleted after the check is done but before the remove() is done, which is probably what happened here.